hs_core/hydroshare/utils.py

```diff
@@ -794,7 +794,13 @@ def prepare_resource_default_metadata(resource, metadata, res_title):
 def get_party_data_from_user(user):
     party_data = {}
     user_profile = get_profile(user)
-    user_full_name = user.get_full_name()
+
+    if user_profile.middle_name:
```

Comment:
Shouldn't this be in the get_full_name method on the AbstractUser model?

Reply:
@sblack-usu AbstractUser model is the default Django auth user model which we have no control over to change, and we are using the default Django user model which does not account for middle name at all in its user model as well as this get_full_name() function. It'd be ideal if we had our own customized user model to start with, but at this point we just have to live with whatever Django default user model provide to us.

Reply:
oh, I didn't realize AbstractUser existed in a django library.  I take back my comment and should have been a bit more thorough looking this over.